src/SFA.DAS.ApplyService.InternalApi/Controllers/FinancialController.cs

```diff
@@ -48,9 +56,9 @@ public async Task<ActionResult> UpdateGrade(Guid applicationId, [FromBody] Finan
         }
 
         [HttpPost("/Financial/{applicationId}/StartReview")]
-        public async Task<ActionResult> StartReview(Guid applicationId)
+        public async Task<ActionResult> StartReview(Guid applicationId, [FromBody] string reviewer)
```

Comment:
Does this work? Normally would have expected a Request object. What if you wanted to expand on it later down the line? Have a look at what I did for Gateway as this was the approach taken within EPAO

Reply:
Updated this, and the assessor review, to use Request objects so consistent with Gateway checks